File: Java/LinearSearch.java
public class LinearSearch {

  public static void main(String[] args) {
  int[] num = {2,3,4,1,6,7};
  int ans = linearSearch(num,5);
  System.out.println(ans);
  }
  
  static int linearSearch(int[] arr,int target){
   if(arr.length==0){
     return -1;
   }
   
   for(int index=0; index<arr.length; index++){
     int elament = arr[index];
     if(elament==target){
       return index;
     }
   }
   return -1;
  }
}

File: Data Structure And Algorithms/linear_search.cpp
#include<iostream>
using namespace std;

int linearSearch(int arr[], int size, int target)
{
    for (int i = 0; i < size; i++)
    {
        if(arr[i] == target)
        {
            cout<<"ELement found at index "<<i<<endl;
            return 0;
        }
    }
    cout<<"Element not found!";
    return 0;
}

int main()
{
    int arr[]={4,-6,1,3,13,7,-1,0,-26};
    int n;
    
    cout<<"Enter the number you want to search:"<<endl;
    cin>>n;

    linearSearch(arr,9,n);

    return 0;
}